src/commands/heroku.rs

```diff
@@ -1,36 +1,131 @@
 use heroku_rs::client::{Executor, Heroku};
 
 use serde::Deserialize;
+use serde_json::Value;
 
 use serenity::framework::standard::{macros::command, Args, CommandResult};
 use serenity::model::prelude::*;
 use serenity::prelude::*;
 
+use crate::authorizations::users::*;
 use crate::config::Config;
 
 #[derive(Debug, Deserialize)]
 struct HerokuApp {
     id: String,
     name: String,
+    released_at: String,
+    web_url: String,
+}
+
+// Get app by name or id
+#[command]
+pub fn get_app(ctx: &mut Context, msg: &Message, mut args: Args) -> CommandResult {
+    let config = bot_config(ctx);
+
+    let app_name = args.single::<String>().unwrap();
```

Comment:
Will the `unwrap()` panic if no argument is provided? If so, I'd prefer an error message instead.